richard juckes:
patch to allow successful build of binutils with gcc-4.0.0.20052020

http://bugs.gentoo.org/show_bug.cgi?id=82907

--- binutils-2.15.92.0.2/gas/config/tc-i386.c
+++ binutils-2.15.92.0.2/gas/config/tc-i386.c
@@ -370,7 +370,7 @@
    prefix), and doesn't work, unless the destination is in the bottom
    64k of the code segment (The top 16 bits of eip are zeroed).  */
 
-const relax_typeS md_relax_table[] =
+const relax_typeS *md_relax_table =
 {
   /* The fields are:
      1) most positive reach of this state,
--- binutils-2.15.92.0.2/gas/config/tc-i386.h
+++ binutils-2.15.92.0.2/gas/config/tc-i386.h
@@ -454,7 +454,7 @@
 
 #define md_operand(x)
 
-extern const struct relax_type md_relax_table[];
+extern const struct relax_type *md_relax_table;
 #define TC_GENERIC_RELAX_TABLE md_relax_table
 
 extern int optimize_align_code;
